docker/prowbazel/bootstrap.py

```diff
@@ -0,0 +1,965 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors.
```

Comment:
We have forked the kubernetest/test-infra repo [here](https://github.com/istio/k8s-test-infra). I have a PR against our fork [here](https://github.com/istio/k8s-test-infra/pull/1). I cleaned up my changes so that they are flags which, when unspecified, do not modify current behavior, but also allow us to modify/turn off the things we need to.